site/_docs/adapter.md

```diff
@@ -236,7 +236,7 @@ They are straightforward to write (you just write a Java class and register it
 in your schema) but do not offer much flexibility in the number and type of
 arguments, resolving overloaded functions, or deriving the return type.
 
-It you want that flexibility, you probably need to write you a
+If you want that flexibility, you probably need to write you a
```

Comment:
Shouldn't we change the second part of the sentence too?
```
you probably need to write you a *user-defined operator*
=>
you probably need to write a *user-defined operator*
```

Reply:
Right.